Formula/sgrep.rb

```diff
@@ -26,7 +26,19 @@ class Sgrep < Formula
 
   uses_from_macos "m4"
 
+  on_arm do
+    # Added automake as a build dependency to update config files for ARM support.
+    depends_on "automake" => :build
+  end
+
   def install
+    if Hardware::CPU.arm?
+      # Workaround for ancient config files not recognizing aarch64 macos.
+      %w[config.guess config.sub].each do |fn|
+        cp Formula["automake"].share/"automake-#{Formula["automake"].version.major_minor}"/fn, fn
+      end
+    end
```

Comment:
This is done in about 10 formulae now. I wonder if we should just include these scripts in `brew`, and perhaps automatically replace any `config.guess` and `config.sub` scripts found in `buildpath`.

Thoughts, @Homebrew/brew?

Reply:
@carlocab Makes sense to me if it would remove this dependency. Otherwise at least a DSL makes sense.